nsqadmin/static/js/views/counter.js

```diff
@@ -0,0 +1,109 @@
+var _ = require('underscore');
+var $ = require('jquery');
+
+var AppState = require('../app_state');
+
+var BaseView = require('./base');
+
+var CounterView = BaseView.extend({
+    className: 'counter container-fluid',
+
+    template: require('./counter.hbs'),
+
+    initialize: function() {
+        BaseView.prototype.initialize.apply(this, arguments);
+        this.listenTo(AppState, 'change:graph_interval', this.render);
+        this.start();
+    },
+
+    start: function() {
+        this.looping = false;
+        this.frameRate = 50;
+        this.targetPollInterval = 30000;
+        this.currentNum = -1;
+        this.tries = 0;
+        this.updateStats();
+        this.startLoop(100);
+    },
+
+    startLoop: function(i) {
+        this.interval = i;
+        this.poller = setInterval(this.updateStats.bind(this), i);
```

Comment:
OK, and related it should probably clear all pending timeouts when this view is removed